package/README.md

```diff
@@ -0,0 +1,178 @@
+# Raiden Services
+
+## What is this repository
+
+This repository contains the documentation and configuration necessary to run the Raiden Services. The services consist of the Raiden Monitoring service (MS) and the Raiden Pathfinding service (PFS).
+
+## TODO
+<!-- **Current release:** [2018.12.0](https://github.com/raiden-network/raiden-transport/tree/2018.12.0) -->
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Requirements](#requirements)
+  - [Hardware](#hardware)
+  - [Software](#software)
+- [Installation](#installation)
+- [Upgrades](#upgrades)
+- [Known issues](#known-issues)
+- [Changelog](#changelog)
+
+## Overview
+
+The Raiden Network uses pathfinding and monitoring services to increase usability.
+To ensure reliability, availability and neutrality it is desirable that those servers are
+being operated by multiple independent entities.
+
+Therefore we provide this repository which allows easy setup of these services.
+It uses docker and docker-compose for easy installation and upgrades.
+
+### Used software
+
+- docker
+- docker-compose
+- Traefik
+
+### Structure
+
+
+```
++-------------------+
+|                   |
+|   Raiden clients  |
+|                   |
++---------+---------+
+          |
+==========|==========
+          |
++---------v---------+
+|                   |
+|      Traefik      |
+|                   |
++---------+---------+
+          |
+          +------------------------+
+          |                        |
++---------v---------+    +---------v---------+
+|                   |    |                   |
+|     Pathfinding   |    |    Monitoring     |
+|                   |    |                   |
++---------+---------+    +---------+---------+
+```
+
+
+We use Traefik as a reverse proxy and also utilize it's capability of automatically provisiong
+Let's Encrypt TLS certificates.
+
+### Network
+
+After a successful deployment the following ports will be in use:
+
+- 80 - HTTP
+  - Redirects to HTTPS
+  - Let's Encrypt HTTP challenge for certificate provisioning
+- 443 - HTTPS
+  - Synapse web and API client access
+  - Metrics export (IP restricted, see below)
+ 
+TODO: check
+
+## Requirements
+
+### Hardware
+
+Minumum recommended for a production setup:
+
+- 2 GiB Ram
+- 2 Cores
+- 10 GiB SSD
+
+### Software
+
+- Docker >= 17.12
+- docker-compose >= 1.21.0
+
+### Other
+
+- A domain (or subdomain) for exclusive use by the services
```

Comment:
per service or for the two services?

Reply:
one domain is enough, we can create subdomains through traefik